Jellyfin.Api/Auth/SyncPlayAccessPolicy/SyncPlayAccessHandler.cs

```diff
@@ -42,10 +47,52 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             var userId = ClaimHelpers.GetUserId(context.User);
             var user = _userManager.GetUserById(userId!.Value);
 
-            if ((requirement.RequiredAccess.HasValue && user.SyncPlayAccess == requirement.RequiredAccess)
-                || user.SyncPlayAccess == SyncPlayAccess.CreateAndJoinGroups)
+            if (requirement.RequiredAccess == SyncPlayAccessRequirementType.HasAccess)
             {
-                context.Succeed(requirement);
+                if (user.SyncPlayAccess == SyncPlayUserAccessType.CreateAndJoinGroups
+                    || user.SyncPlayAccess == SyncPlayUserAccessType.JoinGroups
+                    || _syncPlayManager.IsUserActive(userId!.Value))
```

Comment:
I think a better, albeit more involved and fragile solution is to have the syncplay session id in the route data and use it to verify that the active user is part of _that_ group.

Alternatively, do the check explicitly in the controller eg.
```
if (!_syncPlayManager.IsUserInGroup(userId, groupId))
{
    return Forbid(); // or BadRequest("User cannot alter the state of another group");
}
```